IntegrationTests/IntegrationScenarios/Scenarios/RUM/RUMResourcesScenarioTests.swift

```diff
@@ -28,7 +28,7 @@ class RUMResourcesScenarioTests: IntegrationTests, RUMCommonAsserts {
             ),
             urlSessionSetup: .init(
                 instrumentationMethod: .allCases.randomElement()!,
-                initializationMethod: .allCases.randomElement()!
+                initializationMethod: .afterSDK
```

Comment:
glad you asked, I will clean this logic as a separate PR. Don't want to mix it here.

But the rationale here is, we can't enable the URLSession instrumentation without enabling either RUM or Trace.

This will be included in the public docs too. First either enable RUM or Trace and then enable `URLSesssion` instrumentation.

Reply:
Makes sense 👍. It stands for a clear difference compared to previous API, where `DDURLSessionDelegate()` could have been (legally) constructed before SDK init. Now we don't need `.beforeSDK` nor the entire `InitializationMethod` 🧽👌.